landing-page/pages/index.tsx

```
@@ -50,10 +73,243 @@ const SlotEmoji = ({
+                    ) : (
+                      <Plus
+                        className="text-gray-500 cursor-pointer"
+                        onClick={() => toggleFAQ(index)}
+                      />
+                    )}
+                  </div>
+                </div>
+                {openIndex === index && (
+                  <p
+                    className="mt-2 text-gray-600 flex-1"
+                    dangerouslySetInnerHTML={{ __html: faq.answer }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)